frontend/src/layouts/SideBar/SideBar.vue

```diff
@@ -16,6 +16,7 @@ import { useUserStore } from '@/stores/user';
 const collapsed = useStorage('collapsed', false);
 const toggleMenu = () => (collapsed.value = !collapsed.value);
 const isCollapsed = computed(() => ({ expanded: !collapsed.value }));
+const buttonTitle = computed(() => (collapsed.value ? 'Expand menu' : 'Collapse menu'));
```

Comment:
Nitpick: no need for the brackets around the ternary operator.

Reply:
Fully aware of that but ESLint is enforcing that rule. Will go with ESLint for now.